Flying_Machine.py

```diff
@@ -1,7 +1,15 @@
 from Unit import Unit
+from Attack import Attack
 
 class Flying_Machine(Unit):
     def __init__(self):
-        Unit.__init__(self,200,14.5,'pierce','heavy',2,1,2,0,0,
+        Unit.__init__(self,200,7.5,'siege','heavy',2,1,2.5,0,0,
                       'air',['air'])
+        self.air_attack = 14.5
+        self.air_attack_cooldown = 2
+        self.air_attack_type = 'pierce'
+
+
+    def flying_machine_bombs(self):
+        self.attackable_position.append('ground')
```

Comment:
Whether flying machine can drop bombs or not is a state for each player instead of for each flying machine. Try to ask the owner of the flying machine if the technology has been unlocked during init.